.pipelines/shared-cluster.yml

```diff
@@ -0,0 +1,88 @@
+# No PR triggers as it runs on a schedule
+pr: none
+trigger: none
+
+schedules:
+- cron: "0 1 * * Sun"
+  displayName: ⭕ Rebuild shared cluster Sun 1am (UTC)
+  always: true
+  branches:
+    include:
+    - master
+
+variables:
+  LOCATION: eastus
+  CLUSTER: aro-v4-shared
+
+jobs:
+- job: ⏬ Tear down shared cluster
+  pool:
+    vmImage: "Ubuntu-16.04"
+
+  container:
+    image: mcr.microsoft.com/azure-cli:latest
+
+  steps:
+  - template: ./templates/template-az-cli-login.yml
+    parameters:
+      workingDirectory: /
+      azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
+  - template: ./templates/template-az-aro-extension-prod.yml
+  - displayName: 🧹 Delete cluster
+    script: |
+      az aro delete --yes --resource-group "$CLUSTER" --name "$CLUSTER"
+
+- job: ⏫ Set up shared cluster
+  pool:
+    vmImage: "Ubuntu-16.04"
+
+  container:
+    image: mcr.microsoft.com/azure-cli:latest
+
+  steps:
+  - template: ./templates/template-az-cli-login.yml
+    parameters:
+      workingDirectory: /
+      azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
+  - template: ./templates/template-az-aro-extension-prod.yml
+  - displayName: 🚀 Create RG
+    script: |
+      az group create --name "$CLUSTER" --location "$LOCATION"
+  - displayName: 🚀 Create VNet
+    script: |
+      az network vnet create \
+        --resource-group "$CLUSTER" \
+        --name vnet \
+        --address-prefixes 10.0.0.0/9
```

Comment:
please reuse the existing dev-vnet in v4-eastus - that way we can VPN to the cluster if needs be.